code/modules/clothing/suits/reactive_armour.dm

```diff
@@ -141,8 +142,8 @@
 /obj/item/clothing/suit/armor/reactive/fire
 	name = "reactive incendiary armor"
 	desc = "An experimental suit of armor with a reactive sensor array rigged to a flame emitter. For the stylish pyromaniac."
-	cooldown_message = "<span class='danger'>The reactive incendiary armor activates, but fails to send out flames as it is still recharging its flame jets!</span>"
-	emp_message = "<span class='warning'>The reactive incendiary armor's targeting system begins rebooting...</span>"
+	cooldown_message = span_danger("The reactive incendiary armor activates, but fails to send out flames as it is still recharging its flame jets!")
```

Comment:
are you sure this works in game?

Reply:
yeah I tested it, uses span helpers, so it's nicer to look at, easier to understand, and works the exact same